### DISCUSSION TITLE
QSelect: extra prop no-option-text

### DISCUSSION THREAD

**Discussion by artnova-david** [quasarframework/quasar#9462] [Ideas / Proposals]
When using many q-selects in a form that could potentially have an empty array its a tedious task to provide every 'q-select' with the 'no-option' slot. I know you can create a custom select component that inherits/extends the QSelect component, but for a basic message like '...no options found' it's a bit overkill. ex: I you have 10+ selects on a form it could lead to 50+ lines of code.

```
`<template #no-option>
	<div class="text-grey">
		{{$t("messages.no-options"}}
	</div>
</template>`
```

It would be nice to have an extra prop for example: "**no-options-text**", that show a basic q-item with the text greyed out. If someone needs a more fancy solution, they can still use the #no-options slot to override.

**Comment by ontwikkelfabriek** [comment]
@artnova-david 

I agree. I would advise you to create a `feature request` in the `issues` section instead posting it here in the discussions.

**Reply by artnova-david** [reply]
Makes sense, done!
